blog/_posts/2022-09-01-scala-3.2.0-released.md

```diff
@@ -0,0 +1,244 @@
+- [`scala.Tuple.:*`](https://scala-lang.org/api/3.x/scala/Tuple.html#:*-a82) together with [`scala.Tuple.Append`](https://scala-lang.org/api/3.x/scala/Tuple$.html#Append-0)
+- [`scala.NonEmptyTuple.init`](https://scala-lang.org/api/3.x/scala/NonEmptyTuple.html#init-ffffff96) together with [`scala.Tuple.Init`](https://scala-lang.org/api/3.x/scala/Tuple$.html#Init-0)
+- [`scala.NonEmptyTuple.last`](https://scala-lang.org/api/3.x/scala/NonEmptyTuple.html#last-fffff7c4) together with [`scala.Tuple.Last`](https://scala-lang.org/api/3.x/scala/Tuple$.html#Last-0)
+- [`scala.quoted.Quotes.reflectModule.AppliedTypeModule.apply`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$AppliedTypeModule.html#apply-fffffbf2)
+- [`scala.quoted.Quotes.reflectModule.SymbolMethods.asQuotes`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$SymbolMethods.html#asQuotes-fffffdc2)
+- [`scala.quoted.Quotes.reflectModule.SymbolMethods.termRef`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$SymbolMethods.html#termRef-d26)
+- [`scala.quoted.Quotes.reflectModule.SymbolMethods.typeRef`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$SymbolMethods.html#typeRef-d26)
+- [`scala.quoted.Quotes.reflectModule.TypeReprMethods.substituteTypes`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$TypeReprMethods.html#substituteTypes-3a6)
+- [`scala.quoted.Quotes.reflectModule.TypeReprMethods.typeArgs`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$TypeReprMethods.html#typeArgs-5f)
+- [`scala.quoted.Quotes.reflectModule.TypeTreeModule.ref`](https://scala-lang.org/api/3.x/scala/quoted/Quotes$reflectModule$TypeTreeModule.html#ref-d26)
+- [`scala.Predef.eq`](https://scala-lang.org/api/3.x/scala/Predef$.html#eq-fffff7fa)
+- [`scala.Predef.ne`](https://scala-lang.org/api/3.x/scala/Predef$.html#ne-fffff7fa)
+
+## Should I update to Scala 3.2.0?
+
+We thoroughly tested all release candidates for 3.2.0 to detect any source incompatibilities. We run a so-called Open Community Build, a compilation of nearly 1000 real-life projects on each compiler build. You can read how we achieved this ambitious goal in [the recent blog post](https://virtuslab.com/blog/prevent-scala3-compiler-regressions-community-build/) of our teammate. Thanks to the knowledge we have gathered using Open Community Build, we have managed to ensure that over 98% of projects passing compilation with 3.1.3 are still compiling correctly without any changes to the source code.
+
+In most cases, fixing an error that resulted from source-incompatible change is trivial for the project author. It often only requires explicitly specifying the type in a place where the code relies on type inference. If somehow you encounter source incompatibility that cannot be easily fixed, let us know, and we will try to fix that in upcoming patch releases of the 3.2 compiler.
```

Comment:
[we should not use words like "trivial"](https://docs.scala-lang.org/contribute/inclusive-language-guide.html#the-words-easy-simple-quick-and-trivial) in our communications
```suggestion
In most cases, fixing an error that resulted from source-incompatible change does not require many fundamental steps for the project author. It often only requires explicitly specifying the type in a place where the code relies on type inference. If somehow you encounter source incompatibility that cannot be easily fixed, let us know, and we will try to fix that in upcoming patch releases of the 3.2 compiler.
```

Reply:
Good point. I'm going to be more careful in the future!